lib/sneakers/workers.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module Sneakers
+  # Module to contain the created Worker classes
```

Comment:
I didn't find the issue, but I added a comment that this is probably only a temporary solution and we will want to switch to actual worker class definitions.